README.md

```diff
@@ -63,7 +72,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
```

Comment:
Yes, good answers but I wanted to point out one rule that you did not explicitly mention: every function and only functions create a new local scope. Additionally, scope does not just refer to variables, it includes functions and objects that you have access too as well.